src/routes.tsx

```diff
@@ -36,269 +38,332 @@ enum EmbarkStory {
+      exact: true,
+    },
   },
   {
-    path: localePathPattern + '/new-member/connect-payment/direct',
-    Component: ConnectPaymentsDirectEntry,
-    exact: true,
+    path: `${onboardingLocaleBaseRoute}/connect-payment/direct`,
+    serverRouteData: {
+      titleTextKey: 'ONBOARDING_CONNECT_DD_PAGE_TITLE',
+    },
+    clientRouteData: {
+      Component: ConnectPaymentsDirectEntry,
+      exact: true,
+    },
   },
   {
-    path: localePathPattern + '/new-member/connect-payment/success',
-    Component: TrustlySpinnerPage,
-    exact: true,
+    path: `${onboardingLocaleBaseRoute}/connect-payment/success`,
+    serverRouteData: {
+      titleTextKey: 'ONBOARDING_CONNECT_DD_PAGE_TITLE',
+    },
+    clientRouteData: {
+      Component: TrustlySpinnerPage,
+      exact: true,
+    },
   },
   {
-    path: localePathPattern + '/new-member/connect-payment/fail',
-    Component: TrustlyFailPage,
-    exact: true,
+    path: `${onboardingLocaleBaseRoute}/connect-payment/fail`,
+    serverRouteData: {
+      titleTextKey: 'ONBOARDING_CONNECT_DD_PAGE_TITLE',
+    },
+    clientRouteData: {
+      Component: TrustlyFailPage,
+      exact: true,
+    },
   },
   {
-    path: localePathPattern + '/new-member/connect-payment/retry',
-    Component: TrustlySpinnerPage,
-    exact: true,
+    path: `${onboardingLocaleBaseRoute}/connect-payment/retry`,
+    serverRouteData: {
+      titleTextKey: 'ONBOARDING_CONNECT_DD_PAGE_TITLE',
+    },
+    clientRouteData: {
+      Component: TrustlySpinnerPage,
+      exact: true,
+    },
   },
   {
-    path: localePathPattern + '/new-member/(offer|sign)',
-    Component: OfferNew,
-    exact: true,
+    path: `${onboardingLocaleBaseRoute}/(offer|sign)`,
+    serverRouteData: {
+      titleTextKey: 'OFFER_PAGE_TITLE',
+    },
+    clientRouteData: {
+      Component: OfferNew,
+      exact: true,
+    },
   },
   {
-    path: localePathPattern + '/new-member/sign/success',
-    Component: SignLoading,
-    exact: true,
+    path: `${onboardingLocaleBaseRoute}/sign/success`,
+    serverRouteData: {
+      titleTextKey: '',
+    },
+    clientRouteData: {
+      Component: SignLoading,
+      exact: true,
+    },
   },
   {
-    path: localePathPattern + '/new-member/sign/fail',
-    Component: SignLoading,
-    exact: true,
+    path: `${onboardingLocaleBaseRoute}/sign/fail`,
+    serverRouteData: {
+      titleTextKey: '',
+    },
+    clientRouteData: {
+      Component: SignLoading,
+      exact: true,
+    },
   },
   {
-    path: localePathPattern + '/new-member/(offer|sign)/:id',
-    Component: OfferPage,
-    exact: true,
+    path: `${onboardingLocaleBaseRoute}/(offer|sign)/:id`,
+    serverRouteData: {
+      titleTextKey: 'OFFER_PAGE_TITLE',
+    },
+    clientRouteData: {
+      Component: OfferPage,
+      exact: true,
+    },
   },
   {
-    path: localePathPattern + '/new-member/debugger',
-    Component: Debugger,
-    exact: true,
+    path: `${onboardingLocaleBaseRoute}/checkout/details/:id`,
```

Comment:
Should we keep "offer" in the URL to make more sense which "id" we are referring to?

```suggestion
    path: `${onboardingLocaleBaseRoute}/offer/:id/checkout/details`,
```

Reply:
As I wrote [above](https://github.com/HedvigInsurance/web-onboarding/pull/728#discussion_r778842864) i think the URL didn't look great with `/offer` in it. But as I also wrote, I think the URL parameter should probably be called `cartId` to make it clearer what we're referring to. I went with `id` though, since that's what you call it for the `/(offer|sign)` route.